mltsp/tests/test_flask_app.py

```diff
@@ -2571,6 +2597,7 @@ def test_prediction_page(self):
             res_dict = json.loads(rv.data)
             while "currently running" in fa.check_job_status(res_dict["PID"]):
```

Comment:
What is your question specifically? This is polling to see if the process has finished... which we'll do away with when we implement the new message flow system.